.gitignore

```diff
@@ -1,6 +1,6 @@
-/target
+target/
```

Comment:
why did you change this? This is the default if you do `cargo new` 🤔

Reply:
I wanted to clarify that the target folders inside the test directories were also being included during commits. That's why I made this change to ensure that recursive folders don't accidentally get committed. Hope that is okay!

Reply:
Ok, sure!